Snowdog_Components/components/Molecules/table/_table-variables.scss

```diff
@@ -3,7 +3,9 @@ $table__width                                                : 100% !default;
 $table__padding                                              : 0 !default;
 $table__background                                           : $white !default;
 $table__margin                                               : $spacer 0 !default;
-$table__border                                               : $border-width-base $border-style-base $gray-light !default;
+$table__border-width-base                                    : $border-width-base;
+$table__border-right-width                                   : $border-width-base;
```

Comment:
is this variable only used once in `--tfoot-scope-row`? if so, the name should be more specific. please check for example `// Scope row variant` section and do it similar